### ISSUE TITLE
Squig Herder [Drop that!] Pet Dodging ability

### ISSUE THREAD

**Issue by smingleton** [WarEmu/WarBugs#9842]
![matrik_075](https://user-images.githubusercontent.com/24352165/27458676-1bd44de6-575f-11e7-9c09-7d9058077c5b.jpg)

![matrik_074](https://user-images.githubusercontent.com/24352165/27458841-df8bb026-575f-11e7-9196-d18da570fa32.jpg)

Gas squig is dodging Drop that ability, I was keeping the ability up because of a witch hunter in the area.
I always check my squig to make sure the ability is up/check the duration, even though I know it. 
It wasn't there after I casted so I checked combat log.

**Comment by Torquemadra** [comment]
unable to replicate, ability works fine in tests

**Comment by smingleton** [comment]
![matrik_078](https://user-images.githubusercontent.com/24352165/27471988-90ceeb8e-57ae-11e7-864b-905cdec02cfc.jpg)
![matrik_079](https://user-images.githubusercontent.com/24352165/27471996-97bea43e-57ae-11e7-82de-a50a78a715db.jpg)

Found some screens from earlier on when i first noticed it. This one I was trying to disarm a sword master. 
I got it again after using the ability out of combat trying to replicated the issue. Going to try and replicate in RvR zone, its hard in the heat of battle though to capture right when the pet dodges.

Thank you for the response.
